cypress-io/cypress
# fix: AttemptResult object

- Closes https://github.com/cypress-io/cypress/issues/24110

### Additional details

* Fields are added to AttempResult object
* This change is necessary in order to correspond to the current TestResult properties

### Steps to test

* Create a test with retries
* Log the `currentTestResult`
* Check the object's properties

### PR tasks

* Changed duartion to wallClockDuration
* Changed startedAt to wallClockStartedAt
* Added failedFromHookId
* Added timings
* TODO: Should Timings object be created?
* TODO: Should documentation be updated?

### Questions for reviewers

* Should a Timings object be created?
* Should additional properties be added to AttemptResult object other than those specified in the issue?
<img width="308" alt="image" src="https://user-images.githubusercontent.com/73000452/233867307-84a335aa-6fa4-437c-a1a5-24b24307a104.png">
